tck/src/main/java/jakarta/json/bind/tck/defaultmapping/jsonptypes/JSONPTypesMappingTest.java

```diff
@@ -448,4 +448,24 @@ public void testJsonNumberMapping() {
 
     return; // passed
   }
+
+  /*
+   * @testName: testNullDeserializedToJsonValueNull
+   *
+   * @assertion_ids: JSONB:SPEC:JSB-3.20
+   *
+   * @test_Strategy: Assert that null is properly deserialized to the JsonValue.NULL
+   */
+  @Test
+  public void testNullDeserializedToJsonValueNull() {
+    JsonValueContainer unmarshalledValue = jsonb.fromJson("{ \"instance\" : null }", JsonValueContainer.class);
+    if (!JsonValue.NULL.equals(unmarshalledValue.getInstance())) {
+      fail("Failed to unmarshal null value to the JsonValue.NULL");
+    }
+    unmarshalledValue = jsonb.fromJson("{}", JsonValueContainer.class);
+    if (unmarshalledValue.getInstance() != null) {
```

Comment:
why not simple `Assert.assertNull(...)`?

Reply:
(same few lines above...)

Reply:
For some reason, these assertions are not used anywhere in the tests. I was thinking about doing it the same way for all the tests.

Reply:
I see no reason for not using them - they can have description of the cause/reason. (this wasn't the case for assertTrue/False due to bug in some older junit versions, where the description was ignored)

Reply:
Alright. I will change that.